src/main/scala/higherkindness/rules_scala/common/sbt-testing/Test.scala

```diff
@@ -29,7 +29,12 @@ object TestHelper {
     f: Runner => A
   ) = {
     val options =
-      if (framework.name == "specs2") Array("-ex", scopeAndTestName.replaceAll(".*::", "")) else Array.empty[String]
+      if (framework.name == "specs2") {
+        // Get rid of any extra context before the test name (specs2 doesn't recognize it)
+        val testName = scopeAndTestName.replaceAll("(?<=\\\\Q).*::", "")
```

Comment:
Maybe a raw string would be easier to read here `raw"(?<=\Q).*::"`

Reply:
(it would still need two `\`, as we need to escape the backslash for the regex)